includes/api/class-wc-admin-rest-reports-stock-controller.php

```diff
@@ -0,0 +1,354 @@
+					'readonly'    => true,
+				),
+				'stock_status'   => array(
+					'description' => __( 'Stock status.', 'wc-admin' ),
+					'type'        => 'string',
+					'enum'        => array_keys( wc_get_product_stock_status_options() ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'stock_quantity' => array(
+					'description' => __( 'Stock quantity.', 'wc-admin' ),
+					'type'        => 'integer',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+			),
+		);
+
+		return $this->add_additional_fields_schema( $schema );
+	}
+
+	/**
+	 * Get the query params for collections.
+	 *
+	 * @return array
+	 */
+	public function get_collection_params() {
+		$params                   = array();
+		$params['context']        = $this->get_context_param( array( 'default' => 'view' ) );
+		$params['page']           = array(
+			'description'       => __( 'Current page of the collection.', 'wc-admin' ),
+			'type'              => 'integer',
+			'default'           => 1,
+			'sanitize_callback' => 'absint',
+			'validate_callback' => 'rest_validate_request_arg',
+			'minimum'           => 1,
+		);
+		$params['per_page']       = array(
+			'description'       => __( 'Maximum number of items to be returned in result set.', 'wc-admin' ),
+			'type'              => 'integer',
+			'default'           => 10,
+			'minimum'           => 1,
+			'maximum'           => 100,
+			'sanitize_callback' => 'absint',
+			'validate_callback' => 'rest_validate_request_arg',
+		);
+		$params['exclude']        = array(
+			'description'       => __( 'Ensure result set excludes specific IDs.', 'wc-admin' ),
+			'type'              => 'array',
+			'items'             => array(
+				'type' => 'integer',
+			),
+			'default'           => array(),
+			'sanitize_callback' => 'wp_parse_id_list',
+		);
+		$params['include']        = array(
+			'description'       => __( 'Limit result set to specific ids.', 'wc-admin' ),
+			'type'              => 'array',
+			'items'             => array(
+				'type' => 'integer',
+			),
+			'default'           => array(),
+			'sanitize_callback' => 'wp_parse_id_list',
+		);
+		$params['offset']         = array(
+			'description'       => __( 'Offset the result set by a specific number of items.', 'wc-admin' ),
+			'type'              => 'integer',
+			'sanitize_callback' => 'absint',
+			'validate_callback' => 'rest_validate_request_arg',
+		);
+		$params['order']          = array(
+			'description'       => __( 'Order sort attribute ascending or descending.', 'wc-admin' ),
+			'type'              => 'string',
+			'default'           => 'desc',
+			'enum'              => array( 'asc', 'desc' ),
+			'validate_callback' => 'rest_validate_request_arg',
+		);
+		$params['orderby']        = array(
+			'description'       => __( 'Sort collection by object attribute.', 'wc-admin' ),
+			'type'              => 'string',
+			'default'           => 'date',
+			'enum'              => array(
```

Comment:
These look like the might have been copied from the products endpoint and won't work here?

Reply:
This works here without any trouble, still valid to query products.